README.md

```diff
@@ -42,26 +42,22 @@ export GRAFANA_ADMIN_PASSWORD=password
     -f components/prow/manifests/rbac.yaml -f components/prow/manifests/nginx-controller.yaml
 ```
 
-- Export the nginx-ingress-controller IP address.
+- Export the domain name to an environment variable
```

Comment:
maybe can move it after 
https://github.com/prometheus/prombench/pull/191/files#diff-04c6e90faac2675aa89e2176d2eec7d8R31